### PR TITLE
FEAT: Increase size of HTML Editor

### PR DESCRIPTION
**Pull request by crydotsnake**
## Description

This pull request increases the size of the HTML Editor a bit.

#### Before

Bildschirmfoto 2022-04-02 um 17 39 15

#### After

Bildschirmfoto 2022-04-02 um 17 39 48

The size of the editor wasnt changing automatically, if you added a snippet into it

* [x] I have opened this pull request against ``master``
* [ ] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: djangocms_snippet/templates/djangocms_snippet/admin/change_form.html [hunk 1]
```diff
@@ -19,12 +19,18 @@
 
     // init editor with settings
     var editor = ace.edit(div[0]);
-    editor.setTheme('ace/theme/' + settings.theme);
+    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```
Comment (fsbraun) [inline djangocms_snippet/templates/djangocms_snippet/admin/change_form.html:22]: @crydotsnake I've "played" with dark mode some more and now am proposing a global toggle in the cms toolbar (see https://github.com/django-cms/django-cms/pull/7329). Can I propose to update this media query as follows:


            var darkMode;
            if (window.parent.CMS.API.Toolbar.get_color_scheme) {
                darkMode = window.parent.CMS.API.Toolbar.get_color_scheme() === 'dark';
            } else {
                // django CMS pre-3.11
                darkMode =  window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
            }
            if (darkMode) {
                editor.setTheme('ace/theme/tomorrow_night');
            }  else {
                editor.setTheme('ace/theme/github');
            }

**Review by fsbraun — COMMENTED** [review]
HI @crydotsnake ! 

I've tried to take the dark mode somewhat further. Maybe you can already incorporate this here?

Thanks so much!

**Review by marksweb — CHANGES_REQUESTED** [review]
There's outstanding change requests here without the completed review to flag these.

**Review by fsbraun — CHANGES_REQUESTED** [review]
Hi @crydotsnake ! I‘ve added some comments to adjust to the improved dark mode feature. Would you mind looking at them?

**Pull request closed without merging**